Makefile

```diff
@@ -6,7 +6,7 @@ interuss:
 
 .PHONY: format
 format:
-	clang-format -style=file -i pkg/dssproto/dss.proto
+	clang-format -style=file -i pkg/dss_v1/dss.proto
```

Comment:
nit: Any chance we could avoid `_` in package names?

Reply:
done